TPL/force-app/main/default/lwc/ambulanceRecordsAccount/ambulanceRecordsAccount.js

```diff
@@ -257,7 +257,7 @@ export default class AmbulanceRecordsAccount extends LightningElement {
 
     }
     handleUnassign(){
-        assignAll({currentAccountId:this.recordId,newCaseId:this.selectedCase,currentRecords:this.recordsToDisplay})
```

Comment:
No case will be passed to unassign as it will always clear the assigned case even if the case input field was utilized.